### PR TITLE
Added example CheckWifi1010FirmwareVersion.ino

### PR DESCRIPTION
**Pull request by Rocketct**
Added check firmware version example, CheckWifi1010FirmwareVersion.ino.

@sandeepmistry @facchinm

### CODE REVIEW

File: examples/CheckWifi1010FirmwareVersion/CheckWifi1010FirmwareVersion.ino [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*
+ * This example check if the firmware loaded on the WiFi1010
```
Comment (sandeepmistry) [inline examples/CheckWifi1010FirmwareVersion/CheckWifi1010FirmwareVersion.ino]: I suggest we rename the sketch to `CheckFirmwareVersion`.

As MKR WiFi 1010, MKR Vidor 4000 (and later Uno WiFi Rev2) are supported boards, I suggest you replace `WiFi1010` above with NINA module.

File: examples/CheckWifi1010FirmwareVersion/CheckWifi1010FirmwareVersion.ino [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+void setup() {
+  // Initialize serial
+  Serial.begin(9600);
+  while (!Serial) {
+    ; // wait for serial port to connect. Needed for native USB port only
+  }
+
+  // Print a welcome message
+  Serial.println("WiFi1010 firmware check.");
```
Comment (sandeepmistry) [inline examples/CheckWifi1010FirmwareVersion/CheckWifi1010FirmwareVersion.ino]: Need another print out here, could be `WiFiNINA`?

File: src/WiFi.h [hunk 3]
```diff
@@ -21,6 +21,8 @@
 #ifndef WiFi_h
 #define WiFi_h
 
+#define WIFI_FIRMWARE_LATEST_MODEL "1.1.0"
```
Comment (sandeepmistry) [inline src/WiFi.h]: `_VERSION` instead of `_MODEL` here?

File: examples/CheckWifi1010FirmwareVersion/CheckWifi1010FirmwareVersion.ino [hunk 4]
```diff
@@ -0,0 +1,59 @@
+/*
+ * This example check if the firmware loaded on the WiFi1010
+ * is updated.
+ *
+ * Circuit:
+ * - WiFi1010 board
```
Comment (sandeepmistry) [inline examples/CheckWifi1010FirmwareVersion/CheckWifi1010FirmwareVersion.ino]: MKR WiFi 1010 or MKR Vidor 4000 board?

File: examples/CheckWifi1010FirmwareVersion/CheckWifi1010FirmwareVersion.ino (hunk 1 above)
Comment (Rocketct) [reply examples/CheckWifi1010FirmwareVersion/CheckWifi1010FirmwareVersion.ino]: ok i'll change in all the point required.

File: examples/CheckFirmwareVersion/CheckFirmwareVersion.ino [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    while (true);
+  }
+
+  // Print firmware version on the module
+  String fv = WiFi.firmwareVersion();
+  String latestFv;
+  Serial.print("Firmware version installed: ");
+  Serial.println(fv);
+
+  latestFv = WIFI_FIRMWARE_LATEST_MODEL;
```
Comment (sandeepmistry) [inline examples/CheckFirmwareVersion/CheckFirmwareVersion.ino]: This line should be `WIFI_FIRMWARE_LATEST_VERSION`.

Please also add a line to the Travis CI config to track build errors for the new sketch: https://github.com/arduino-libraries/WiFiNINA/blob/master/.travis.yml#L43

**Pull request closed without merging**